lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -1067,7 +1067,10 @@ def get(self, request, subsection_id):
                 'possible_graded': original_grade.possible_graded,
             }
         except PersistentSubsectionGrade.DoesNotExist:
-            grade_data = self._get_grade_data_for_not_attempted_assignment(user_id, usage_key)
+            try:
+                grade_data = self._get_grade_data_for_not_attempted_assignment(user_id, usage_key)
+            except ValueError:
+                grade_data = None
 
         results = SubsectionGradeResponseSerializer({
             'original_grade': grade_data,
```

Comment:
Can the gradebook MFE deal with a case where `original_grade` is null?

Reply:
no, it will only work with None